addon/components/head-layout.js

```diff
@@ -15,15 +15,20 @@ export default Component.extend({
    */
   shouldTearDownOnInit: true,
 
-  headElement: computed(function() {
-    let documentService = getOwner(this).lookup('service:-document');
-    return documentService.head;
-  }),
```

Comment:
Is this removed because it is not needed anymore?

Reply:
This was moved to `init`. Some tests is overwriting this and it was causing the computed clobbering deprecation.